batchflow/models/tf/deeplab.py

```diff
@@ -39,8 +39,6 @@ def default_config(cls):
         config['body/embedding'] = dict(base=aspp)
 
         config['body/decoder/upsample'] = dict(layout='b')
-
-        config['head'] = dict(layout='c', kernel_size=1)
```

Comment:
please comment on this

Reply:
Previously, it was OK to have non-empty layout here. Now it is passed to `TFModel.head` directly and thus needs `filters`. This parameters (layout and kernel size) are now part of `EncoderDecoder.head`